app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -86,7 +90,7 @@ class ExplorationActivityTest {
   @Test
   fun testExploration_toolbarTitle_isDisplayedSuccessfully() {
     getApplicationDependencies(TEST_EXPLORATION_ID_30)
-    ActivityScenario.launch<ExplorationActivity>(createExplorationActivityIntent(TEST_EXPLORATION_ID_30)).use {
+    ActivityScenario.launch<ExplorationActivity>(createExplorationActivityIntent(internalProfileId, TEST_TOPIC_ID_0, TEST_STORY_ID_0, TEST_EXPLORATION_ID_30)).use {
```

Comment:
Use imports for ActivityScenario.

Reply:
+1

Reply:
Updated here and everywhere else in this.